content/_guides/catalyst.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Catalysing and De-escalation
+category: chanop
+---
+
+People often use Libera.Chat to get help with something on their computer.
+Technical support channels are one of the things we're here for!
+
+If you have ever had a technical problem, you know how overwhelming it can be.
+Sometimes people seeking help will get very frustrated and might have a temper
+or be rude. Patient and efficient help can be vital in avoiding conflict in
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
```

Comment:
I have critique on this part. This doesn't address services occassionally being down during which new ops cannot be reveived and doesn't communicate to people who don't already know how to get op attention.

https://mikaela.info/blog/english/2015/04/01/keep-the-ops-opped.html

Reply:
Additionally Matrix users have additional difficulties seeing the op status of users as the current instance of the bridge only syncs ops to the other side and they need to use +O for syncing state as much as they can.

Reply:
we try to warn users ahead of services downtime and people can (usually) check chanserv flags to find ops. the social benefit of not wearing far outweighs these points

Reply:
(additionally, I don't want to make compromises on productively handling users for the sake of a matrix shortcoming)

Reply:
Clients have a lot of differences in how they show globals or wallops, so they can be missed and it's possibly for ops to just not be on between the time of scheduled downtime announcement and beginning.

Reply:
downtime is typically short, find network staff if you missed the wallops and have an urgent problem. it seems like a one in a million problem and certainly shouldn't discard a tried and true aspect of nonescalation

Reply:
Sorry, I forgot in my previous comment, could there be a link somewhere on this section to another page explaining how to get connection to an operator whose flags aren't visible (including `/msg chanserv flags #channel` and how to connect e.g. account Mikaela to Aminda using information available through ChanServ or `/who`?

Reply:
channels without viewable access lists typically don't want their ops found for whatever reason, I think it would be irresponsible to advise people on how to get around that desire

Reply:
I am asking about channels with public access list. 

Using a real life example of a channel which has a compromise solution by only having a bot opped:

* There is spam on ##jrpg (and in this case I am pretending it's a much larger channel than it actually is)
* New IRC user doesn't know who are ops and how to reach them
* More advanced IRC user knows to /msg chanserv flags ##jrpg list
* The access list contains account name `Mikaela`. However currently that nickname doesn't exist on the channel. How will the user know that `Mikaela` is actually using name `Aminda`?

If there is no page explaining this issue, should I open a new issue to request one to be created which could then be linked from here?

Reply:
We won't be able to handle every single potential edge case, and it is recommendations, not fixed rule. We think that the benefits of not being opped all the time are bigger than the issues and caveats, channel operators disagreeing are free to wear their op hat.  On finding channel ops, it has advantages and disadvantages, e.g. if there are three people wearing an op hat but all of them are afk, that is not helpful for people being new to IRC either, it is actually potentially frustrating. 

tl;dr: we know there are downsides to each variant, but we prefer this one, thus we put it in our recommendation and I think the wording is fine as is.

Reply:
the point about instructions on querying `CS FLAGS` is a reasonable one though

Reply:
I think it creates more edge cases (private versus public) and additional documentation needs (which flag is what, how do I reach someone who is not online, how do I find someone by account instead of nick) that will make it a lot harder to read, so if, then it should imho be split into separate documentation and linked.

Reply:
yes, Mikaela suggested a separate document

Reply:
@emersonveenstra (in https://github.com/Libera-Chat/libera-chat.github.io/issues/60#issuecomment-901961470) points out that it's documented in https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted so could that be linked somewhere here?

Reply:
As Fuchs points out above, we still need additional documentation which we'll then link to from various places. One specific page we need is a brief walk-through on advising what flags are actually needed for non-founder ops when setting up a new channel. One of the last pages i did, hurriedly with the intention of rewriting it before launch, was https://libera.chat/guides/creatingchannels which is a bit dire really and could do with explaining how to set up ops in it too. How to check your ops would be part of the checklist for that and probably an ideal place to link to from this.

Until that's done, we have already mentioned how to view flags on https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted but it's not an ideal place to throw people from this document in particular.

Reply:
Actually on second thoughts, a faq section of its own might be better, and have that faq section pointed to from the getting unbanned instructions. since pointing people seeking ops for a problem situation might not find it encouraging to be punted to instructions on how a banned person could undermine their efforts.